portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,25 +45,35 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
+    comments.jsonList.clear();
+
     for (Entity entity : results.asIterable()) {
       String fromDatastore = (String) entity.getProperty("title");
-      if (!output.contains(fromDatastore)) {
-        output.add(fromDatastore);
+      if (!comments.jsonList.contains(fromDatastore)) {
+        comments.jsonList.add(fromDatastore);
       }
     }
-
+    
```

Comment:
remove unnecessary spaces

Reply:
let me know if I didnt do this, im not 100% sure but i think i did